### DISCUSSION TITLE
Laser no longer burns any material (was working before latest update)

### DISCUSSION THREAD

**Discussion by Actionslats** [arkypita/LaserGRBL#1041] [Q&A]
Hi All. im having an issue that my 2.5W laser no longer burns. it produces the laser (blue in colour). within the 'config' window on lasergrbl i had changed the speed setting to match the laser output (2500). this was before the update. None of the settings or config appear to have changed, and im now at a loss as to what to do. Any and all ideas welcome. TIA

**Comment by Actionslats** [comment]
the laser is also as focused as it was when it was working too

**Comment by gmmanonymus111** [comment]
<del>Since updating LaserGRBL shouldn't cause any problems like that, make sure your laser isn't dead.
Or, if you think the update broke something, you can downgrade by downloading an older version from [GitHub releases](https://github.com/arkypita/LaserGRBL/releases)
EDIT:
You could try cleaning your laser's lens.</del>

> within the 'config' window on lasergrbl i had changed the speed setting to match the laser output (2500)

Change it back to 255. It has nothing to do with your laser's power, it only mudulates the TTL signal, so your laser can be set between 0 and 2.5W (if you had a 5W laser, it could be set between 0 and 5W). It actually regualtes how much voltage your laser gets from the 12V source.

**Comment by Actionslats** [comment]
I have now tried everything mentioned and more. Iv bought a new lense to eliminate that from the process. 
![BD2DCE68-B775-49E5-AFEC-63FC46AA0959](https://user-images.githubusercontent.com/66518999/88086715-b79c9980-cb7f-11ea-87e7-faa25cf5c314.jpeg)
This is how the laser projects with no lense. Is this correct or is the diode knackered?

**Comment by Actionslats** [comment]
Photo taken Parallel to laser housing/top rail

**Comment by StuartB4** [comment]
That's pretty normal. Laser dots are actually rectangular. You just need to focus to the smallest you can get, it will never be a round dot though.

**Comment by arkypita** [comment]
and this is why: [https://www.youtube.com/watch?v=NbHnxL44eXo](https://www.youtube.com/watch?v=NbHnxL44eXo)

**Comment by StuartB4** [comment]
More people should see that video, would help them understand a bit more.
